runtime/src/main/java/org/corfudb/util/MetricsUtils.java

```diff
@@ -47,39 +61,63 @@
+     * property to enable the metrics collection.
+     * <li> metricsJmxReportingEnabled: Boolean taken from jvm corfu.metrics.jmxreporting
+     * property to enable jmx reporting of metrics.
+     * <li> metricsJvmCollectionEnabled: Boolean taken from jvm corfu.metrics.jvm
+     * property for enabling reporting on jmv metrics such as garbage collection, threads,
+     * and memory consumption.
+     * <li> metricsLogAnalysisEnabled: Boolean taken from jvm corfu.metrics.log.analysis
+     * property for enabling reporting on logger statistics.
+     * <li> metricsLogInterval: Integer taken from jvm corfu.metrics.log.interval
+     * property for enabling and setting the intervals of reporting to log output (in
+     * seconds). A positive value indicates the reporting is enabled at provided
+     * intervals.
+     * <li> metricsCsvInterval: Integer taken from jvm corfu.metrics.csv.interval
+     * property for enabling and setting the intervals of reporting to csv (in seconds).
+     * A positive value indicates the reporting is enabled at provided intervals.
+     * <li> metricsCsvFolder: String taken from jvm corfu.metrics.csv.folder
+     * property for destination path of csv reporting.
      * </ul>
      *
-     * <p>This function will be called to set the value of metricsCollectionEnabled,
-     * metricsJmxReportingEnabled, and metricsLogInterval for reporting. The value
-     * of metricsCollectionEnabled represents the expected collection status.
-     * Slf4j reporting is enabled if metricsLogInterval is set to a positive integer
-     * representing emission intervals in seconds.
-     * Jmx reporting is enabled if metricsJmxReportingEnabled is set to {@code true}.
+     * <p>This method will be called to set the value of above-mentioned properties
+     * for collection and reporting. For example using the following jvm flags will
+     * enable collection of corfu, jvm, and log statistics and their reporting through
+     * logs, csv, and jmx.
+     *
+     * {@code -Dcorfu.metrics.collection=True
+     * -Dcorfu.metrics.csv.interval=30
+     * -Dcorfu.metrics.csv.folder=/tmp/csv5
+     * -Dcorfu.metrics.jmxreporting=True
+     * -Dcorfu.metrics.log.analysis=True
+     * -Dcorfu.metrics.jvm=True
+     * -Dcorfu.metrics.log.interval=60}
      */
     private static void loadVmProperties() {
         metricsCollectionEnabled = Boolean.valueOf(System.getProperty(PROPERTY_METRICS_COLLECTION));
+
         metricsJmxReportingEnabled = Boolean.valueOf(System.getProperty(PROPERTY_JMX_REPORTING));
+        metricsJvmCollectionEnabled = Boolean.valueOf(System.getProperty(PROPERTY_JVM_METRICS_COLLECTION));
+        metricsLogAnalysisEnabled = Boolean.valueOf(System.getProperty(PROPERTY_LOG_ANALYSIS_COLLECTION));
+
         try {
             metricsLogInterval = Long.valueOf(System.getProperty(PROPERTY_LOG_INTERVAL));
             metricsSlf4jReportingEnabled = metricsLogInterval > 0 ? true : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1125)